Snakefile

```diff
@@ -60,6 +60,21 @@ rule prepare_conv_pp:
     shell:
         "python scripts/prepare_conv_pp.py {input.opsd} {input.gpkg} {input.b3_regions} {input.scalar_template} {output}"
 
+
+rule prepare_heat_demand:
+    input:
+        raw="raw/",
```

Comment:
```suggestion
        raw="raw/ERA5_weatherdata",
```

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/76/commits/f2b37f8082de1b48b5c269947c5a8171703cf94b. I used `weatherdata` instead of `ERA5_weatherdata` since any other weather data could be used and we only use ERA5 weather data for now but aim for Merra2 data.